<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicFocusStage</class>
 <widget class="QWidget" name="BasicFocusStage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>382</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Focus Stage</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Focus Stage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAbort">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../visao.qrc">
     <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="StatusConnected">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>not connected</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="StatusCmode">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>NONE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCmode">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Control:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonTakeLocal">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>31</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Take Local Control</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxOverride">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>63</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>override</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPosition_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Current Position:</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentPosition">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>0.0000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPower">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Power:</string>
   </property>
  </widget>
  <widget class="QLabel" name="powerState">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>UNK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPositionUnits">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>um</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="cModeFrame">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>180</y>
     <width>731</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabSpacing">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>137</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Move to:</string>
    </attribute>
    <widget class="QPushButton" name="ButtonNeg">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPosition">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonPos">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelOffset_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Absolute:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditOffset">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonGO">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelOffset">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Move by:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>137</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Advanced</string>
    </attribute>
    <widget class="QPushButton" name="ButtonHomeNeg">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>55</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Home Negative</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonHomePos">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Home Positive</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonEnable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusEnable">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Disabled</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonHome">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkFront">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Negative Limit</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBack">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Positive Limit</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkHome">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Home Switch</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="statusOther">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>other status</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonGOPreset">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>131</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Go To Preset</string>
   </property>
  </widget>
  <zorder>title</zorder>
  <zorder>ButtonAbort</zorder>
  <zorder>StatusConnected</zorder>
  <zorder>StatusCmode</zorder>
  <zorder>LabelCmode</zorder>
  <zorder>ButtonTakeLocal</zorder>
  <zorder>checkBoxOverride</zorder>
  <zorder>LabelPosition_3</zorder>
  <zorder>LabelPower</zorder>
  <zorder>powerState</zorder>
  <zorder>labelPositionUnits</zorder>
  <zorder>cModeFrame</zorder>
  <zorder>toolBox</zorder>
  <zorder>statusOther</zorder>
  <zorder>currentPosition</zorder>
  <zorder>ButtonGOPreset</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEditPosition</tabstop>
  <tabstop>ButtonAbort</tabstop>
  <tabstop>lineEditOffset</tabstop>
  <tabstop>ButtonPos</tabstop>
  <tabstop>ButtonNeg</tabstop>
  <tabstop>checkBoxOverride</tabstop>
  <tabstop>ButtonTakeLocal</tabstop>
 </tabstops>
 <resources>
  <include location="../visao.qrc"/>
 </resources>
 <connections/>
</ui>
